src/main/resources/swagger.api/examples/errors.sample

```diff
@@ -0,0 +1,15 @@
+{
+  "errors": [
+    {
+      "message": "must not be null",
+      "type": "-1",
+      "code": "Validation error",
```

Comment:
Type has number but code has string. Feels a bit the opposite.

Reply:
I'll removed outdated functionality and ErrorType enum, because I copied them by mistake